cmd/test_util.go

```diff
@@ -112,3 +116,29 @@ func assertEqualYaml(fileToFix string, fileFixed string, auditFunc func(resource
 	assert.Nil(err)
 	assert.Equal(correctlyFixedResources[0], fixedResource)
 }
+
+// WriteToTmpFile writes a single resource to a tmpfile, you are responsible
+// for closing the file afterwards, that's why the function returns the file
```

Comment:
Just nitpicking, do you mean deleting the file? It looks like it is closed on line 139 `err = tmpfile.Close()`